source/features/comment-on-draft-pr-indicator.tsx

```diff
@@ -5,8 +5,13 @@ import * as pageDetect from 'github-url-detection';
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 import createBanner from '../github-helpers/banner.js';
+import {getDiscussionAuthor, getUsername} from '../github-helpers/index.js';
 
 function addDraftBanner(newCommentField: HTMLElement): void {
+	if (getDiscussionAuthor() === getUsername()) {
```

Comment:
I'd append this to the asLongAs list, where it must be preceded. Add comment specifying that it must come after the isDraftPR check (also in asLongAs) or else it's executed on every page